### ISSUE TITLE
[Bug] Navbar has visibility issue in dark mode

### ISSUE THREAD

**Issue by StephanosNikitis** [IndieHub25/GroqTales#191]
## Describe the bug

When we hover on "CONNECT WALLET" button in dark mode it turns completely black, therefore nothing is visible.

## Steps to Reproduce

1. Go to 'https://www.groqtales.xyz/'
2. Hover on "CONNECT WALLET" button in dark mode

## Expected behavior

when hovering on the "connect wallet" button during dark mode it should change to white color not black so that it is visible.

## Screenshots
When hovered:
<img width="1897" height="96" alt="Image" src="https://github.com/user-attachments/assets/929fa2be-67b8-4470-be6c-20b84ca9969e" />

When not hovered:
<img width="1896" height="108" alt="Image" src="https://github.com/user-attachments/assets/b838a90c-746f-4173-94fe-1811974d1691" />

## Environment

- OS/Device: Windows, macOS, Linux
- Browser: All

---

**Related Links:**

- [GroqTales Website](https://www.groqtales.xyz/)

---

Please assign me this issue under SWOC'26.

**Comment by Drago-03** [comment]
@StephanosNikitis Okay you may start working on this
